lib/dmv.rb

```diff
@@ -9,8 +10,12 @@ def add_facility(facility)
   end
 
   def facilities_offering_service(service)
-    @facilities.find do |facility|
-      facility.services.include?(service)
+    facilities_offering_service = []
+    @facilities.find_all do |facility|
+      if facility.services.include?(service)
```

Comment:
Since services is an attribute of the facility class, this functionality is better suited to live there.